apps/studio/src/features/editing-experience/components/CreateCollectionPageModal/PreviewLayout.tsx

```diff
@@ -57,7 +58,7 @@ export const PreviewLayout = (): JSX.Element => {
             justify="center"
             whiteSpace="pre"
           >
-            {`You're previewing a ${currentType === "link" ? "PDF " : ""}collection page`}
+            {`You're previewing a ${currentType === ResourceType.CollectionLink ? "collection link" : "collection page"}`}
```

Comment:
nit: can move the `collection` part out
```suggestion
            {`You're previewing a collection ${currentType === ResourceType.CollectionLink ? "link" : "page"}`}
```

Reply:
done